app/components/noteworthy-and-popular-project/component.js

```diff
@@ -0,0 +1,26 @@
+import { computed } from '@ember/object';
+import Component from '@ember/component';
+
+export default Component.extend({
+    contributors: computed('project', function() {
+        const contribs = this.get('project.contributors.content.canonicalState');
+        if (!contribs) {
+            return;
+        }
+        const len = contribs.length;
+        const namePath = index => this.get(`project._internalModel.__relationships.initializedRelationships.contributors.canonicalState.${index}.__data.links.relationships.users.data.attributes.family_name`) || this.get(`project._internalModel.__relationships.initializedRelationships.contributors.canonicalState.${index}.__data.links.relationships.users.data.attributes.given_name`);
+        switch (len) {
+        case 1: return namePath(0);
+        case 2: return `${namePath(0)}, ${namePath(1)}`;
+        case 3: return `${namePath(0)}, ${namePath(1)}, ${namePath(2)}`;
+        default: return `${namePath(0)}, ${namePath(1)}, ${namePath(2)} +${len - 3}`;
```

Comment:
It's unfortunately not the same :/ with `and` instead of `,`. Could make it into a component with a `useAnd` argument?

Reply:
Oh, I see, that's annoying. I guess `useAnd` would be fine, but the inconsistency on one page seems wrong. We don't use "and" in any other list of contributors that I can find, so I'd argue we should just do commas and make our UX incrementally better in passing.

Also, looking around just now I noticed you've swapped these compared to prod. If we're not changing anything, new/noteworthy/popular should use "and":
<img width="271" alt="screen shot 2018-01-23 at 9 34 59 am" src="https://user-images.githubusercontent.com/6776190/35281418-d6b1212e-0020-11e8-9243-bd96420f2564.png">
and the main dashboard list should use just commas:
<img width="223" alt="screen shot 2018-01-23 at 9 35 13 am" src="https://user-images.githubusercontent.com/6776190/35281440-e1a6b2b0-0020-11e8-8dd1-5c541b479b1e.png">

Reply:
Hold on just a minute and I will make Product choose the future direction for lists of contributors.

Reply:
But assume it will be one thing regardless, and we'll come down on a "and or no and" side before it launches.

Reply:
Totally just doing this actually, good call

Reply:
The actual dashboard does the same thing on both lists (I didn't think so, but it was a difference in the number of contributors per section so I didn't see it). If you have few enough to list all of them (i.e. 2-3), then you put in the and (with a comma for 3 and not for 2). If you have too many, then it's commas-only with the +.

Reply:
Oh, huh. Consistent logic for inconsistent display. I guess that's okay.

Reply:
It would probably be better to do, ", and 53 more." so that we lose a branch or two, but that might be annoying on smaller columns. For example:

![screen shot 2018-01-23 at 12 59 02 pm](https://user-images.githubusercontent.com/6599111/35292120-85204bb0-003d-11e8-89be-2febff87bc02.png)
![screen shot 2018-01-23 at 12 59 43 pm](https://user-images.githubusercontent.com/6599111/35292121-852a9ec6-003d-11e8-9fe1-0449d0304b30.png)

Verifying with Sara… she says, "heck yes". So do that, please.

Reply:
ahhhh